eo2js/src/commands/transpile.js

```diff
@@ -78,12 +78,23 @@ const needsRetranspile = function(source, transpiled) {
  */
 const transform = function(tojo, options, transformations, parser) {
   const text = fs.readFileSync(tojo[verified]).toString()
-  let xml = parser.parse(text)
+  let xml;
+  try {
+    xml = parser.parse(text);
+  } catch (e) {
+    throw new Error(`Failed to parse XML for ${tojo[verified]}: ${e.message}`);
+  }
   const pth = pathFromName(xml['program']['@_name'])
   const isTest = hasMeta(xml, 'tests')
   const transpiled = path.resolve(options.target, dir, `${pth}.xmir`)
   const dest = path.resolve(options.project, `${pth}${isTest ? '.test' : ''}.js`)
-  if (needsRetranspile(tojo[verified], transpiled)) {
+  if (!needsRetranspile(tojo[verified], transpiled)) {
+    if (options.verbose) {
+      console.log(`Skipping ${pth} - already transpiled`);
+    }
+    return;
```

Comment:
@2IMT we're trying to avoid multiple return statements, here's why: https://www.yegor256.com/2015/08/18/multiple-return-statements-in-oop.html